sample_files/.travis.yml

```diff
@@ -66,9 +68,17 @@ env:
 virtualenv:
   system_site_packages: true
 
+# Enable this if your repository needs sass command (needed for website related repos)
+#before_install:
+#  - rvm install ruby --latest
+#  - gem install bootstrap-sass
+#  - gem install compass --pre
+
 install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  # Enable this if your repositoru needs sass command (needed for website related repos)
+  #- sudo apt-get install ruby-sass  
```

Comment:
This isn't needed either, dependency is installed in `bootstrap-sass` into Ruby env